src/types.d.ts

```diff
@@ -66,3 +66,46 @@ export interface ClaimInterface {
   terminus_pool_id: number;
   title: string;
 }
+
+type DropDBData = {
```

Comment:
Can we split these into a separate types file? It's not bad now, but it's easy to see this file growing without much cohesion.